outlines/understanding-graphql-introspection-system.md

```diff
@@ -0,0 +1,37 @@
+# Understanding GraphQL's Introspection System
```

Comment:
How is this different than what we already have here https://tailcall.run/graphql/graphql-introspection/